README.md

```diff
@@ -35,7 +35,8 @@ secrets from Conjur.
 
 This module requires that you have:
 - Puppet v5 _or equivalent EE version_
-- Puppet v5 agent on the nodes
+- Puppet v6 _or equivalent EE version_ ([Community level](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community)
```

Comment:
Can we say more about what community means here?  Even just a few words, along with the link?  I really want to avoid confusion.